packages/azure-functions/Shared/reference-helpers.js

```diff
@@ -4,7 +4,7 @@ const randomReferenceString = (length) => {
   const chars = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ'
   let result = ''
   for (let i = 0; i < length; i++) {
-    const randomIndex = Math.floor(randomBytes(1)[0] / 255 * chars.length)
+    const randomIndex = Math.floor(randomBytes(1)[0] / 256 * chars.length)
```

Comment:
Good spot Stewart! Paul suggested instead using `crypto.randomInt(0, chars.length)`. I agree that is a simpler and better approach - it should give us a more random result.

Reply:
Thanks John, will do